client/src/components/LandingNavbar/useStyles.ts

```diff
@@ -2,7 +2,19 @@ import { makeStyles, Theme } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
-    flexGrow: 1,
+    padding: '0 2rem',
+    '@media (max-width: 900px)': {
+      paddingLeft: 0,
+      paddingRight: 0,
+    },
+    '@media (max-width: 600px)': {
+      position: 'relative',
+    },
```

Comment:
Material UI offers breakpoints that can be adjusted in the theme file. By convention, that is typically preferred. I would recommend to convert to utilizing the breakpoints the theme provides and adjust those in the theme file accordingly.

Refer to https://material-ui.com/customization/breakpoints/

Reply:
Will do that.